docs/docusaurus.config.js

```diff
@@ -65,6 +65,26 @@ const config = {
 					src: 'img/logo.svg',
 				},
 				items: [
+					{
+						to: '/parachains/picasso-parachain-overview',
+						position: 'left',
+						label: 'Networks',
+					  },
+					{
+						to: '/products/centauri/Dotsama-ibc',
```

Comment:
should not just be dotsama IBC - cross ecosystem IBC 

And then list specifically the progress of each ecosystem  - format should be: 

-architecture 
-integrations 
-etc